checkov/sca_package/runner.py

```diff
@@ -134,15 +134,17 @@ def find_scannable_files(
         files: list[str] | None,
         excluded_paths: set[str],
         exclude_package_json: bool = True,
-        excluded_file_names: set[str] | None = None
+        excluded_file_names: set[str] | None = None,
+        extra_supported_package_files: set[str] | None = None
     ) -> set[Path]:
         excluded_file_names = excluded_file_names or set()
+        extra_supported_package_files = extra_supported_package_files or set()
         input_paths: set[Path] = set()
         if root_path:
             input_paths = {
                 file_path
                 for file_path in root_path.glob("**/*")
-                if file_path.name in SUPPORTED_PACKAGE_FILES and not any(p in file_path.parts for p in excluded_paths)
+                if file_path.name in SUPPORTED_PACKAGE_FILES.union(extra_supported_package_files) and not any(p in file_path.parts for p in excluded_paths)
```

Comment:
the default of `extra_supported_package_files` should be `set()`, not `None`,
it will cause an error.

please cover it also in your tests

Reply:
1.its covered in tests, since we already have tests without this new parameter. 
2. the default cant be mutable, it fails in lint test
3. in line 141 if extra_supported_package_files==None it will be updated to set()